packages/welldone-wallet/src/lib/nep413.ts

```diff
@@ -43,16 +43,9 @@ export const payloadSchema = new Map([
   ],
 ]);
 
-export const isNep413Message = (message: any): boolean => {
-  if ("message" in message && "recipient" in message && "nonce" in message) {
-    return true;
-  }
-  return false;
-};
-
 export const serializeNep413 = (
   signMessageParams: SignMessageParams
 ): Buffer => {
   const payload = new Payload({ ...signMessageParams });
-  return Buffer.from(serialize(payloadSchema, payload));
+  return Buffer.from(serialize(payloadSchemaNep413, payload));
 };
```

Comment:
Yes, I think that's a great idea, this would avoid potential human errors when others implement `signMessage` .

Can we expose it on this PR I think we could add it in this file:
https://github.com/near/wallet-selector/blob/main/packages/core/src/lib/helpers/verify-signature/payload.ts

Reply:
Thank you for your positive consideration of my suggestion.

I've added `serializeNep413` in that file, please check it out.